src/Mono.Android/Android.Runtime/Timing.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+
+namespace Android.Runtime
+{
+	public class Timing : IDisposable
```

Comment:
Should we have doc-comments on this?

I'm also not thrilled with `Timing`, because the class *itself* doesn't provide timing info.  Instead, the timing info is written to `adb logcat`.

I can't think of a better name either. `LoggingTimer`, perhaps?